addons/website_blog/models/website_blog.py

```diff
@@ -22,6 +22,13 @@ class BlogBlog(models.Model):
     ]
     _order = 'name'
 
+    def _default_sequence(self):
+        blog = self.search([], limit=1, order='sequence DESC')
+        if blog:
+            return blog.sequence + 5
+        return 10000
```

Comment:
What do you think about this suggestion?
```suggestion
    def _default_sequence(self):
        self.env.cr.execute('SELECT MAX(sequence) FROM %s' % self._table)
        max_sequence = self.env.cr.fetchone()[0]
        if max_sequence:
            return max_sequence + 5
        return 10000
```
Like in https://github.com/odoo/odoo/pull/126637/commits/440ac1c8694faebd1a82ba76b07a27c3ee9d2323#diff-82a6d79184f3ddf90cec5e020b736fa1080ad900d1754e21a8b1d2d4f55d897bR26-R40

And maybe add a comment to explain the need of 10000 magical number :)

Reply:
> @SergeBayet @divy-odoo I think we should rather investigate how the other apps do this... I doubt it's anything like this PR suggestion. Would not it work to just set the same sequence to all (16 is the default in some models I think), review the `_order` value of the model to use the sequence and that's it?

@qsm-odoo @divy-odoo 
If we set default `sequence` to 16, for example, and use `_order = 'sequence, id'` to ensure that the most recently created blog appears at the end of the list, this will only work initially. Once the blogs are reordered using the 'handle', the sequence values will increment (e.g., 16, 17, 18). A new blog created afterward will default back to a sequence of 16, which will cause it to be positioned near the beginning of the list, instead of at the end as expected (see the task).

When I was looking for the same pattern in the code base, I was searching for `return 10000` instead of `_default_sequence` and indeed there is a better way to do that. like :
```python
    _order = 'sequence'

    def _default_sequence(self):
        return (self.search([], order="sequence desc", limit=1).sequence or 16) + 1

    sequence = fields.Integer("Sequence", default=_default_sequence)
```
With this `_default_sequence` definition, we ensure that the last blog created will be the last in the list.

Reply:
@SergeBayet Done changes, Thanks.